### ISSUE TITLE
Migrate can't create new database

### ISSUE THREAD

**Issue by steebchen** [prisma/migrate#332]
Creating a new database with

```
yarn prisma2 migrate save --create-db --name init --experimental
yarn prisma2 migrate up --experimental
```

using the schema

```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

// The `datasource` block is used to specify the connection to your DB.
// Set the `provider` field to match your DB type: "postgresql", "mysql" or "sqlite".
// The `url` field must contain the connection string to your DB.
// Learn more about connection strings for your DB: https://pris.ly/connection-strings
datasource db {
  provider = "sqlite" // other options are: "mysql" and "sqlite"
  url      = "sqlite:./dev.db"
}
// Other examples for connection strings are:
// SQLite: url = "sqlite:./dev.db"
// MySQL:  url = "mysql://johndoe:johndoe@localhost:3306/mydb"
// You can also use environment variables to specify the connection string: https://pris.ly/prisma-schema#using-environment-variables

// By adding the `generator` block, you specify that you want to generate Prisma's DB client.
// The client is generated by runnning the `prisma generate` command and will be located in `node_modules/@prisma` and can be imported in your code as:
// import { PrismaClient } from '@prisma/client'
generator client {
  provider = "prisma-client-js"
}

// Next steps:
// 1. Add your DB connection string as the `url` of the `datasource` block
// 2. Run `prisma2 introspect` to get your data model into the schema (this will override this file and delete all comments!)
// 3. Run `prisma2 generate` to generate Prisma Client JS
// 4. Start using Prisma Client JS in your application

model User {
	id    String  @default(cuid()) @id
	email String  @unique
	name  String?
}
```

results in 

```
Failure during a migration command: Generic error. (error: The datasource db already exists in this Schema. It is not possible to create it once more.)
```

using the latest alpha (prisma2@2.0.0-alpha.655, binary version: 321a958cfb891344dca5b7948c030e7273835565).

**Comment by tomhoule** [comment]
https://github.com/prisma/prisma-engine/pull/428 should have fixed it

**Comment by pantharshit00** [comment]
It works fine in the latest stable version `preview020.3`:
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/22195362/73252070-222bd880-41e0-11ea-911b-df65f6ec1a44.png">


But it was broken in alpha, @tomhoule has fixed it

**Comment by pantharshit00** [comment]
Fixed in `prisma2@2.0.0-alpha.656, binary version: c2994e6e2511c80d995a5756a92ee9c12df808a5`
